src/Tribe/Capabilities.php

```diff
@@ -15,24 +47,64 @@ class Tribe__Events__Capabilities {
 			'delete_private_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'author' => array( // full permissions for content the user created
+		],
+		//Full permissions for content the user created.
+		'author'      => [
 			'read',
 			'edit_posts',
 			'edit_published_posts',
 			'delete_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'contributor' => array( // create, but not publish
+		],
+		//Create, but not publish.
```

Comment:
:no_entry_sign: No space found before comment text; expected "// Create, but not publish." but found "//Create, but not publish."

<sub>hash: 7296f688ea94f4568644a804f579250a</sub>